appveyor.yml

```diff
@@ -0,0 +1,31 @@
+environment:
+  matrix:
+    - PYTHON: "C:\\Python27"
+    - PYTHON: "C:\\Python34"
+    - PYTHON: "C:\\Python35"
+    - PYTHON: "C:\\Python36"
+    - PYTHON: "C:\\Python36-x64"
+
+install:
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  # We need wheel installed to build wheels
+  - "python.exe -m pip install wheel"
+  # lxml 4.1.1 has binary release for Windows
+  - "python.exe -m pip install lxml==4.1.1"
```

Comment:
I wasn't sure if there was a better way to do this. Did a quick search and found https://gist.github.com/pombredanne/72130ee6f202e89c13bb but that might be overkill

Reply:
Well it's possible to use this directly in requirements.txt:

```
lxml==4.1.1; platform_system == 'Windows'
```

The question is whether it's better approach than specifying it for tests...

Reply:
I have been running Weblate on Windows, and I can confirm there is a few hoops like this required. This particular one might be more for avoiding a manual compile that requires a large toolchain and matching Python VC++ tools install

Reply:
Anyway it seems they are consistently doing binary releases now, so it's probably best to remove this completely.